kratos/processes/fast_transfer_between_model_parts_process.h

```diff
@@ -88,14 +83,16 @@ class KRATOS_API(KRATOS_CORE) FastTransferBetweenModelPartsProcess
      * @todo Add intermediate combinations of constraints, conditions and elements
      */
     enum class EntityTransfered {
-        NODES = 0,
-        ELEMENTS = 1,
-        NODESANDELEMENTS = 2,
-        CONDITIONS = 3,
-        NODESANDCONDITIONS = 4,
-        CONSTRAINTS = 5,
+        NODES               = 0,
+        ELEMENTS            = 1,
+        NODESANDELEMENTS    = 2,
+        CONDITIONS          = 3,
+        NODESANDCONDITIONS  = 4,
+        CONSTRAINTS         = 5,
         NODESANDCONSTRAINTS = 6,
-        ALL = 7
+        GEOMETRIES          = 7,
+        NODESANDGEOMETRIES  = 8,
```

Comment:
We're adding these but there is no transfer of geometries in the cpp implementations (maybe I didn't find it).

Reply:
I am adding for the other PR, it makes no harm, and will reduce the number of changes in the other PR